### DISCUSSION TITLE
GPT-4的反轉魔咒

### DISCUSSION THREAD

**Discussion by sorn32** [1101-GS4719-C/Python-Community-GS4719-C#1643] [General]
在文章中提到了一本名為《代數思維》的書，其中作者發現了早期多層神經網路在自由泛化普遍關係上的失敗，並給出了原則性的理由，來預測這些架構失敗的原因——在許多現實問題中，你永遠不可能完全覆蓋可能的示例空間，而在像LLM這樣缺乏明示變數和變數操作的大量數據驅動型系統中，當你嘗試推斷出訓練示例空間之外的情況時，你就沒戲了。

如今大型語言模型在訓練過程中存在推理能力缺失，知道A=B，卻無法反推B=A。
![反轉魔咒](https://github.com/NCU-GS4719-Python/Python-Community/assets/76047120/f5d6db05-9dd1-463e-b393-d151cdfcfdf0)

除此之外，你們在使用ChatGPT時，有遇到過特別的情況嗎?

資料出處:
https://redian.news/wxnews/596566
https://www.techbang.com/posts/109880-gpt-4-is-also-difficult-to-escape-the-reverse-curse-new-study

**Comment by davidteng00000** [comment]
我認為可能是因為ChatGPT的訓練資料都是由網路抓出來的，由於湯姆克魯斯的媽媽可能沒有那麼知名，所以要透過湯姆克魯斯做連結GPT才能搜尋到
